.travis.yml

```diff
@@ -1,19 +1,29 @@
 language: objective-c
 osx_image: xcode11.1
-env:
-  - MODE=OSX CFG=Debug
-  - MODE=OSX CFG=Release
-  - MODE=iOS CFG=Debug
-  - MODE=iOS CFG=Release
-  - MODE=tvOS CFG=Debug
-  - MODE=tvOS CFG=Release
-  - MODE=watchOS CFG=Debug
-  - MODE=watchOS CFG=Release
-script:
-  - ./.travis.sh "${MODE}" "${CFG}"
-notifications:
-  email: false
+jobs:
+  include:
+    - stage: Swift PM
+      script:
+      # Build the project using Swift PM
+        - swift build
+      # Test the project using Swift PM
+        - swift test
 
-branches:
-  except:
-    - google
+    - stage: Xcode Tests
+      env:
```

Comment:
What about updating **.travis.sh** to be like: `{iOS|OSX|tvOS|SwiftPM} {Debug|Release|Both}`?, and then the Swift tests will be added to the rest of the test suite.

Reply:
That can also work, but the logic in the rest of the script will be a bit more complex because it can't simply waterfall thru to building up an `xcodebuild` command line; but something like that should be possible.

If you want to just do the simple travis.yml change for now, we can look at updating the script later to get debug and release coverage for SwiftPM also.